recipes/easy_profiler/all/test_package/example.cpp

```diff
@@ -0,0 +1,306 @@
+//#define FULL_DISABLE_PROFILER
```

Comment:
Could you use a smaller example? The idea is checking the package, not exercising the library.

Reply:
Sure. I just grabbed an example from the original project. Which is usually what I do for the packages I create. It seemed to be okay for the other packages I have created.